Gemfile

```diff
@@ -48,3 +48,7 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
+
```

Comment:
you should remove empty line between gems

Reply:
And push them to the top. Production gems should be the first thing you see